lib/vsc/mympirun/mpi/intelmpi.py

```diff
@@ -227,17 +230,17 @@ def mpiexec_set_global_options(self):
 
         if 'I_MPI_DEVICE' in self.mpiexec_global_options:
             del self.mpiexec_global_options['I_MPI_DEVICE']
-        if not 'I_MPI_FABRICS' in self.mpiexec_global_options:
+        if 'I_MPI_FABRICS' not in self.mpiexec_global_options:
             self.mpiexec_global_options['I_MPI_FABRICS'] = self.device
 
-        scalable_progress = (self.mpitotalppn * self.nruniquenodes) > 64
+        scalable_progress = (self.mpiprocesspernode * self.nruniquenodes) > 64
```

Comment:
We shouldn't hard code 64. It should a global variable at the minimum. What exactly does this do?